src/context/toast.tsx

```diff
@@ -4,16 +4,26 @@ import Toast, { Variant } from "@leafygreen-ui/toast";
 import { WordBreak } from "components/Typography";
 import { TOAST_TIMEOUT } from "constants/index";
 
-export type ToastProps = {
+type ToastProps = {
```

Comment:
No need to expose this since it was used internally. It also wasn't used anywhere.